content/en/docs/components/pipelines/author-a-pipeline/pipelines.md

```diff
@@ -0,0 +1,141 @@
++++
+title = "Pipelines"
+description = "Create a pipeline"
+weight = 5
++++
+
+A *pipeline* is a description of a multi-task workflow, including how tasks relate to each other to form an computational graph. Pipelines may have inputs which can be passed to tasks within the pipeline.
+
+## Author a pipeline
+Unlike components which have three authoring approaches, pipelines have one authoring approach: they are defined using Python pipeline functions decorated with `@dsl.pipeline()`. For example:
```

Comment:
nit: with the latest changes. it can be `@dsl.pipeline` (doesn't need to be called).